File: trail1/출석 부르기 2/calling-attendance-2.c
#include <stdio.h>

int main() {
    while (1) {
        int N;
        scanf("%d", &N);
        if (N == 1) {printf("John\n");}
        else if (N == 2) {printf("Tom\n");}
        else if (N == 3) {printf("Paul\n");}
        else if (N == 4) {printf("Sam\n");}
        else {printf("Vacancy\n"); break;}
    }
    return 0;
}

File: trail1/출석 부르기 2/calling-attendance-2.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        while(true) {
            int N = int.Parse(Console.ReadLine());
            if (N == 1) {Console.WriteLine("John");}
            else if (N == 2) {Console.WriteLine("Tom");}
            else if (N == 3) {Console.WriteLine("Paul");}
            else if (N == 4) {Console.WriteLine("Sam");}
            else {Console.WriteLine("Vacancy"); break;}
        }
    }
}
